app/views/scripts/components/html/sideMenu.html

```diff
@@ -0,0 +1,56 @@
+<!-- MENU -->
```

Comment:
Maybe we could use this component for all views that have the sidebar, making its content optional between this and the back button?

Reply:
Good thinking! I'll refactor it tomorrow 👍